gui-react/src/containers/SettingsContainer/ResetSettings/ResetSettings.tsx

```diff
@@ -0,0 +1,81 @@
+import { useTheme } from 'styled-components'
+import { RowFlex, DiscardWarning } from '../styles'
```

Comment:
```suggestion
import { RowFlex } from '../styles'
```
tiny nit pick but `DiscardWarning` isn't being used anywhere